switchapi/switch_nhop_int.h

```diff
@@ -100,7 +100,6 @@ extern "C" {
   do {                                                                    \
     SWITCH_MEMSET(&_nhop_key, 0x0, sizeof(switch_nhop_key_t));            \
     if (_api_nhop_info->nhop_type == SWITCH_NHOP_TYPE_IP) {               \
-      SWITCH_ASSERT(SWITCH_RIF_HANDLE(_api_nhop_info->rif_handle));       \
```

Comment:
shouldn't here be an OR case, instead of removing the check?
SWITCH_ASSERT(SWITCH_RIF_HANDLE(_api_nhop_info->rif_handle) || SWITCH_LAG_HANDLE(_api_nhop_info->rif_handle))

Reply:
Addressed.